L/LoopTools/bundled/wrappers/looptools.h

```diff
@@ -0,0 +1,303 @@
+#if 0
```

Comment:
What are these files in `bundled/wrappers/` needed for?  They're unused.

Reply:
> What are these files in `bundled/wrappers/` needed for? They're unused.

Thank you very much! I'm pretty sorry for messing up the things. they should be from some experiments I did when I first tried to wrapping over this package... please feel free to remove them.

Reply:
Ok, no worries, I removed them.

Reply:
Hallo, I still failed to make it work on my Mac M1. The error is `Library not loaded: @rpath/libquadmath.0.dylib `, using Julia for macOS ARM. It worked for macOS x86 via Rosetta. Any idea making it work? Which version of gcc should be installed? Thank you for your efforts!

Reply:
What version of Julia?

Reply:
I tried both macOS ARM (M-series Processor) [64-bit](https://julialang-s3.julialang.org/bin/mac/aarch64/1.7/julia-1.7.2-macaarch64.dmg) (experimental) and [julia-1.8.0-beta1-full.tar.gz](https://github.com/JuliaLang/julia/releases/download/v1.8.0-beta1/julia-1.8.0-beta1-full.tar.gz). LoopTools_jll can be added and precompiled. The error arose if I `using LoopTools_jll`

Reply:
problem solved by installing gfortran from [gfortran 12-experimental for ARM, macOS 12 (Monterey)](https://github.com/fxcoudert/gfortran-for-macOS/releases/tag/12-arm-alpha) and `ln -s /usr/local/gfortran/lib/libquadmath.0.dylib /usr/local/lib/libquadmath.0.dylib`